src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -155,7 +197,7 @@ hooks.addPureHook('POST', 'resin', 'device', {
 });
 
 hooks.addPureHook('PATCH', 'resin', 'device', {
-	POSTRUN: async (args) => {
+	PRERUN: async (args) => {
```

Comment:
Why did you change this to a `PRERUN`?

Reply:
We need to get the existing appId _before_ we make any changes in order to delete service installs with from that app only.  My understanding is that once we get to POSTRUN, it's too late for that.

Reply:
Ok, it makes sense that you now need to know the before state in order to only delete those